jekyll/_cci2/github-integration.adoc

```diff
@@ -328,6 +328,14 @@ Key functionality enabled through the GitHub App integration includes the follow
 [#installing-github-app-alongside-github-oauth-app]
 ### Installing GitHub App alongside the GitHub OAuth app
 
+[CAUTION]
+====
+Having both a GitHub App trigger *and* a GitHub OAuth trigger configured in the same project could result in duplicate builds. To prevent this behavior, make sure to disable the GitHub OAuth trigger. To do so:
```

Comment:
```suggestion
Having both a GitHub App trigger *and* a GitHub OAuth trigger configured in the same project could result in duplicate builds. To prevent this behavior, make sure to disable the GitHub OAuth trigger. To do so, follow these steps:
```